composer.json

```diff
@@ -96,6 +97,24 @@
       "delete_vendor_files": true,
       "include_modified_date": false,
       "include_author": false
+    },
+    "changelogger": {
+      "changelog": "changelog.md",
+      "types": {
+        "feat" : "Feature",
+        "tweak" : "Tweak",
+        "fix" : "Fix",
+        "performance" : "Performance",
+        "security" : "Security",
+        "accessibility" : "Accessibility",
+        "compatibility" : "Compatibility",
+        "deprecated" : "Deprecated",
+        "language" : "Language"
+      },
+      "formatter": {
+        "filename": "bin/class-tec-changelog-formatter.php"
```

Comment:
We will make similar changes to ET on Friday. But currently ET and common gets TEC's title in the `.md` file. We will need to provide specific to plugin formatter which they would extend TEC's and overwritte TEC's `$title` property after we make it `protected` instead of `private`